README.md

```diff
@@ -317,6 +317,20 @@ plugins:
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins-ci.org/secure/Dashboard.jspa?selectPageId=17346) for known compatibility issues.
 
+## Triggering Configuration Reload
+
+You have the following option to trigger a configuration reload:
+
+- via the user interface: `Manage Jenkins -> Configuration -> Reload existing configuration`
+- via http POST to `JENKINS_URL/configuration-as-code/reload`
+  Note: this needs to include a valid CRUMB and authentication information e.g. username + token of a user with admin
```

Comment:
Not true since Jenkins 2.96. If you use an API token, you don't need a crumb.

Reply:
we're gonna bump jenkins version in jcasc very soon so makes sense to make the statement valid for Jenkins >= 2.96

Reply:
no need to include the valid crumb section, just say username + api token imo
```suggestion
  Note: this needs to authenticated e.g. username + api token with admin
```